test/core/sequencer/benchmark.js

```diff
@@ -14,7 +14,7 @@ test('benchmark all modules', function(t) {
 
   console.log('############ Benchmarks ############');
 
-  var sequencer = ImageSequencer({ ui: false, inBrowser: false });
+  var sequencer = ImageSequencer({ ui: false, inBrowser: false,useWasm:true });
```

Comment:
Yes, like couldn't we copy this whole line `test('benchmark all modules', function(t) {` to `test('benchmark all modules with WebAssembly', function(t) {` and then have it say `console.log('############ Benchmarks with WebAssembly ############');` instead? That could be a more standard way to do this, now that we have the `useWasm` flag.

Reply:
Making few changes here!!